src/components/Button.js

```diff
@@ -0,0 +1,14 @@
+import { PropTypes } from 'prop-types';
+import Button from 'react-bootstrap/Button';
+
+function renderButton({ name = 'btn', color = 'light' }) {
```

Comment:
## Enforce the use of `ES6` syntax
- [ ] Kindly change this function declaration into an arrow function.